packages/client/src/Components/RfiStatusBoardChannel.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
@sanjayradadiya  - this component doesn't add any functionality to RFIStatusBoard. 

It just passes the data through.

We don't need it, do we?

Reply:
we are passing all states related data through this component so we make it separate from other channel components.

Instead of maintaining all logic inside factory helper we can delegate some part here.

Reply:
Do we need this component though?  Can't we just use RfiStatusBoard in factory.tsx?